essentials-command-helper/src/main/kotlin/io/github/nyliummc/essentials/CommandBuilder.kt

```diff
@@ -0,0 +1,89 @@
+package io.github.nyliummc.essentials
```

Comment:
Unsure if this is should be considered as an impl detail and put in an impl package